screens/Profile/MyProfileScreen/styles.ts

```diff
@@ -1,13 +1,57 @@
-import { StyleSheet } from 'react-native'
+import { StyleSheet } from "react-native"
+import { black, lightPeach, peach } from "../../../assets/colors/colors"
 
 export const styles = StyleSheet.create({
+  button: {
+    alignItems: "center",
+    backgroundColor: lightPeach,
+    borderColor: peach,
+    borderRadius: 10,
+    borderWidth: 2,
+    height: 38,
+    justifyContent: "center",
+    marginLeft: 14,
+    width: 90,
+  },
+  buttonText: {
+    alignItems: "center",
+    fontSize: 12,
+    fontWeight: "bold",
+    justifyContent: "center",
+    margin: 1,
+  },
```

Comment:
Having a button with `height` and `width` properties as pixels is not flexible (consider different font scaling). You should not specify `height` at all, but use `padding` for `buttonText`. Concerning `width` you could either not specify it at all or use `%` sizing if you want to have multiple buttons of strictly the same `width`.

Reply:
Hey I modified the styles of MyProfileScreen and also slightly the ones of ExpandableDescription and GeneralProfile trying to use proportions and not use negative margin. I also updated the demo video because it slightly changed the display. Tell me what you think of it now !

Reply:
Perfect!